src/components/landing/Landing.tsx

```diff
@@ -0,0 +1,46 @@
+import React, {useState} from 'react';
+import { Link } from 'react-router-dom';
+import './Landing.css';
+import terminal from './landing-images/terminal.gif';
+
+const Landing: React.FC = () => {
+    const [display, setDisplay] = useState('block');
```

Comment:
Should have a type for useState
`const [display, setDisplay] = useState<string>('block')`